### DISCUSSION TITLE
git-link breaks on GitLab branches

### DISCUSSION THREAD

**Discussion by jgarte** [sshaw/git-link#103] [Q&A] [answered]
Has anyone else experienced `git-link` giving `404` GitLab urls if not on the default branch?

**Comment by sshaw** [comment]
Hi, I use this on GitLab repos quite frequently and have not experienced any issues. 

What version are you using and are you setting `git-link-default-branch` or, `git-link.branch` via `git`? 

With version 0.8.3 I set `git-link-default-branch` to another branch and did not have any issues. Is it possible that the file you were linking to did not exist on that branch?

**Comment by sshaw** [comment, marked as answer]
> With version 0.8.3 I set git-link-default-branch to another branch and did not have any issues...

Just tried on the latest release (0.8.6) and it worked too
